cmd/influx/cli/cli.go

```diff
@@ -182,7 +182,13 @@ func (c *CommandLine) Run() error {
 	for {
 		select {
 		case <-c.osSignals:
-			close(c.Quit)
+			// Attempt to gracefully exit
```

Comment:
This isn't needed since this section is single threaded. The close should always succeed. If the channel is already closed, then the `Quit` channel will always be the one chosen by `select`.